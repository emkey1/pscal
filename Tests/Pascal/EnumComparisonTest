program EnumComparisonTest;
type
  Color = (Red, Green, Blue);
var
  a, b: Color;
begin
  a := Red;
  b := Green;
  if a = Red then writeln('Color a is Red');
  if a <> b then writeln('Enums compare not equal');
  if b > a then writeln('Enums compare greater');
end.
