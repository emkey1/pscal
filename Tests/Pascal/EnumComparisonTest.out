Color a is Red
Enums compare not equal
Enums compare greater
